src/Team.jsx

```diff
@@ -1,19 +1,171 @@
+import Modal from 'react-modal';
+import { Row, Col }  from 'react-bootstrap';
+import axios from 'axios';
+import Select from 'react-select';
 
-function Skills(props) {
-  return (
-    <div>
-      <h1>The Team</h1>
-      <ul>
-        <li>Brian</li>
-        <li>Andrew</li>
-        <li>Jordan</li>
-        <li>Jeff</li>
-        <li>Dane</li>
-        <li>Joel</li>
-      </ul>
-    </div>
-  );
+import AddTeamMemberForm from './AddTeamMemberForm.jsx';
+import TeamMemberDisplay from './TeamMemberDisplay.jsx';
+import DeleteModal from './DeleteModal.jsx';
+
+var api = (process.env.REACT_APP_API);
+
+class Team extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      teamMembers: [],
+      addModalIsOpen: false,
+      deleteModalIsOpen: false,
+      selectedTeamMember: {
+        id: "",
+        name: "",
+        title: "",
+      }
+    };
+
+    this.fetchTeamMembers = this.fetchTeamMembers.bind(this);
+    this.openAddModal = this.openAddModal.bind(this);
+    this.closeAddModal = this.closeAddModal.bind(this);
+    this.openDeleteModal = this.openDeleteModal.bind(this);
+    this.closeDeleteModal = this.closeDeleteModal.bind(this);
+    this.onSelectChange = this.onSelectChange.bind(this);
+    this.addTeamMember = this.addTeamMember.bind(this);
+    this.shouldDelete = this.shouldDelete.bind(this);
+  }
+
+  openAddModal() {
+    this.setState({ addModalIsOpen: true });
+  }
+
+  closeAddModal() {
+    this.setState({ addModalIsOpen: false });
+  }
+
+  openDeleteModal() {
+    this.setState({ deleteModalIsOpen: true });
+  }
+
+  closeDeleteModal() {
+    this.setState({ deleteModalIsOpen: false });
+  }
+
+  fetchTeamMembers() {
+    axios.get(`${api}/teammembers/`)
+      .then((result) => {
+        const teamMembers = result.data.slice();
+        this.setState({
+          teamMembers: teamMembers
+        });
+      })
+      .catch((err) => {
+        console.log(`Error fetching team members: ${err}`);
+      });
+  }
+
+  onSelectChange(obj) {
+    axios.get(`${api}/teammembers/${obj.id}`)
+      .then((result) => {
+        const teamMemberData = result.data;
+        this.setState({
+          selectedTeamMember: {
+            id: teamMemberData.id,
+            name: teamMemberData.name,
+            title: teamMemberData.title,
+          }
+        });
+      }).catch((err) => {
+        console.log(`Error: ${err}`);
+      });
+  }
+
+  addTeamMember(name, title) {
+    axios.post(`${api}/teammembers/`, {
+      name: name,
+      title: title
+    }).then((response) => {
+        this.closeAddModal();
+        this.fetchTeamMembers();
+      }).catch(err => {
+        console.log(`Error: ${err}`);
+      });
+  }
+  shouldDelete(response) {
+    if (response) {
+      this.deleteTeamMember();
+    }
+    this.closeDeleteModal();
+  }
+
+  deleteTeamMember() {
+    axios.delete(`${api}/teammembers/${this.state.selectedTeamMember.id}`)
```

Comment:
I like the syntax a lot more because it is more readable. There are many places where we do string concatenation but I think that readability would be enhanced if we started to use string interpolation more